paasta_tools/cli/cmds/spark_run.py

```diff
@@ -82,6 +83,16 @@ def add_subparser(subparsers):
         default='default',
     )
 
+    list_parser.add_argument(
+        '-w', '--work-dir',
+        default='%s:%s' % (os.getcwd(), DEFAULT_SPARK_WORK_DIR),
+        help=(
+            "The read-write volume to mount in format "
+            "local_abs_dir:container_abs_dir, defaults "
+            "to $PWD:/spark_drvier."
```

Comment:
Do we already use a default printing argparse?
https://stackoverflow.com/questions/12151306/argparse-way-to-include-default-values-in-help

If not, this would be a fancier way to do this